Planning guide: Hide Katello parts in non-Katello (#679)
theforeman/foreman-documentation: guides/doc-Planning_Guide/topics/Introduction.adoc

[[chap-Architecture_Guide-Introduction_to_Red_Hat_Satellite]]
== Introduction to {ProjectName}

_{ProjectName}_ is a system management solution that enables you to deploy, configure, and maintain your systems across physical, virtual, and cloud environments.
{Project} provides provisioning, remote management and monitoring of multiple {RHEL} deployments with a single, centralized tool.
ifdef::katello::[]
_{ProjectName} Server_ synchronizes the content from Red{nbsp}Hat Customer{nbsp}Portal and other sources, and provides functionality including fine-grained life cycle management, user and group role-based access control, integrated subscription management, as well as advanced GUI, CLI, or API access.
endif::[]

ifdef::katello::[]
_{ProjectName} {SmartProxyServer}_ mirrors content from {ProjectName} Server to facilitate content federation across various geographical locations.
Host systems can pull content and configuration from {SmartProxyServer} in their location and not from the central {ProjectServer}.
endif::[]
ifndef::katello::[]

> Host systems can pull configuration from {SmartProxyServer} in their location and not from the central {ProjectServer}.

Reviewer: Should this bit can only pull ? That's how the sentence reads. 

Do you mean can also - meaning it is a benefit - or can only - so you need to pay attention

Author: I copied the line that was there (12) and removed "content and" to keep it simple.

In Puppet it's certainly pull and it can be a benefit to pull from a more local system. Imagine you have your Satellite in New York and a Smart Proxy in your Amsterdam DC. Then it is a benefit that all agents in Amsterdam do not need to reach out to New York.

Nowadays we also have push (REX), which we should mention here as well but I considered that out of scope. However, there the same benefits are true: a more local deployment means fewer round trips across the ocean.

As for the can also vs only: it's also. You can still deploy a Smart Proxy without, for example, Puppet and reach out to the one on the Foreman server.

Reviewer: ```suggestion
Host systems can also pull configuration from {SmartProxyServer} in their location rather than the central {ProjectServer}.
```
I understand now. 
Up to you to decide whether to update this, since this is out of scope as you just dropped the content word from the above sentence and was using this for another scenario.

endif::[]
{SmartProxyServer} also provides localized services such as Puppet server, DHCP, DNS, or TFTP.
{SmartProxyServer}s assist you in scaling your {Project} environment as the number of your managed systems increases.

{SmartProxyServer}s decrease the load on the central server, increase redundancy, and reduce bandwidth usage.
For more information, see xref:chap-Documentation-Architecture_Guide-Capsule_Server_Overview[].

> ifdef::katello::[]

Reviewer: This excludes the entire section from Satellite docs.
